add: Canpack doc page (#3547)
dfinity/portal: docs/developer-docs/developer-tools/dev-tools-overview.mdx

- Ruby [`ic_agent` by Terry.Tu](https://github.com/tuminfei/ic_agent)

### Canpack

> - [Canpack](/docs/current/developer-docs/developer-tools/off-chain/canpack): A tool used to better facilitate communication between canisters written in different languages; currently supports importing Rust crates into Motoko canisters.

Reviewer: Rephrased for precision:

```suggestion
- [Canpack](/docs/current/developer-docs/developer-tools/off-chain/canpack): A tool used to facilitate communication between canisters written in different languages; currently supports calling Rust crates from Motoko canisters.
```
